File: code-coach/free/easy/popsicles.java
import java.util.Scanner;

public class Program {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int siblings = input.nextInt();
        int popsicles = input.nextInt();

        if (popsicles > 0 && popsicles % siblings == 0) {
            System.out.println("give away");
        } else {
            System.out.println("eat them yourself");
        }
    }
}

File: code-coach/free/easy/popsicles.cs
using System;

namespace Sololearn {
    class Program {
        static void Main(string[] args) {
            int siblings, popsicles;
            siblings = Convert.ToInt32(Console.ReadLine());
            popsicles = Convert.ToInt32(Console.ReadLine());

            // your code goes here
            if (popsicles > 0 && popsicles % siblings == 0) {
                Console.WriteLine("give away");
            }
            else {
                Console.WriteLine("eat them yourself");
            }
        }
    }
}
